.circleci/config.yml

```diff
@@ -76,7 +76,8 @@ workflows:
           filters:
             branches:
               only:
-                - /^.*main.*/
+                - ^main$
+                - /^patch\/.+/
```

Comment:
This would still only release when there is a bump of `MARKETING_VERION` on `Common.xcconfig`, right? I think that is correct, just double checking.

Also, wondering about the name, not sure if it wouldn't be better to use `hotfix/fix-something` just so it is clear this is due to something wrong in a released version 🤔 In the end though, this name is minor since the more important is the functionality 🙂

Reply:
Correct @lucaschifino , the logic to trigger them remains the same.
----
With regards to utilize `hotfix/fix-something` , that is the goal yeah.
Basically the `patch/base_{some_version}` will be the base branch to make PRs to. The single hotfixes (as we suppose to be having tickets assigned to them) could follow the `hotfix/MOB-0000...` or directly the `MOB-0000` we used.
This way we'll get two birds with one stone being able to review them and possibly to open PRs from the same branches into `main`. Or cherry pick them.